### DISCUSSION TITLE
first-time-setup.yaml problem

### DISCUSSION THREAD

**Discussion by w0016** [greenelab/lab-website-template#177] [Q&A] [answered]
Hello,
I am trying to set up my site and I am working on step 3. which is manually running "first-time-setup" under "Actions" tab. And it fails every time both when I try "fork" and "use this template". What steps should take in order to fix this problem?

Thank you

**Comment by vincerubinetti** [comment, marked as answer]
I'm assuming you're talking about this run:
https://github.com/w0016/w16.github.io/actions/runs/4664051685/jobs/8255937463

```
Run rm -rf * .*
rm: refusing to remove '.' or '..' directory: skipping '.'
```

This is a bug on my part. I will release a hotfix today. If you want a quick fix for this, you can change the `run: rm -rf * .*` line in `first-time-setup.yaml` to `run: rm -rf *`.

**Reply by vincerubinetti** [reply]
Here is the pull request if you're interested in the full fix:
https://github.com/greenelab/lab-website-template/pull/178
